libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParserVisitor.ts

```diff
@@ -43,178 +43,178 @@ import { KeyContext } from "./ExpressionAntlrParser";
 	 * labeled alternative in `ExpressionAntlrParser.primaryExpression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitFuncInvokeExp?: (ctx: FuncInvokeExpContext) => Result;
+    visitFuncInvokeExp?: (ctx: FuncInvokeExpContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by the `indexAccessExp`
 	 * labeled alternative in `ExpressionAntlrParser.primaryExpression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitIndexAccessExp?: (ctx: IndexAccessExpContext) => Result;
+    visitIndexAccessExp?: (ctx: IndexAccessExpContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by the `unaryOpExp`
 	 * labeled alternative in `ExpressionAntlrParser.expression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitUnaryOpExp?: (ctx: UnaryOpExpContext) => Result;
+    visitUnaryOpExp?: (ctx: UnaryOpExpContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by the `binaryOpExp`
 	 * labeled alternative in `ExpressionAntlrParser.expression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitBinaryOpExp?: (ctx: BinaryOpExpContext) => Result;
+    visitBinaryOpExp?: (ctx: BinaryOpExpContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by the `primaryExp`
 	 * labeled alternative in `ExpressionAntlrParser.expression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitPrimaryExp?: (ctx: PrimaryExpContext) => Result;
+    visitPrimaryExp?: (ctx: PrimaryExpContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by `ExpressionAntlrParser.file`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitFile?: (ctx: FileContext) => Result;
+    visitFile?: (ctx: FileContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by `ExpressionAntlrParser.expression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitExpression?: (ctx: ExpressionContext) => Result;
+    visitExpression?: (ctx: ExpressionContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by `ExpressionAntlrParser.primaryExpression`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitPrimaryExpression?: (ctx: PrimaryExpressionContext) => Result;
+    visitPrimaryExpression?: (ctx: PrimaryExpressionContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by `ExpressionAntlrParser.stringInterpolation`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitStringInterpolation?: (ctx: StringInterpolationContext) => Result;
+    visitStringInterpolation?: (ctx: StringInterpolationContext) => Result;
 
-	/**
+    /**
 	 * Visit a parse tree produced by `ExpressionAntlrParser.textContent`.
 	 * @param ctx the parse tree
 	 * @return the visitor result
 	 */
-	visitTextContent?: (ctx: TextContentContext) => Result;
+    visitTextContent?: (ctx: TextContentContext) => Result;
```

Comment:
We should fix this indentation for the rest of the comment block, each star symbol (*) should be aligned with the first one for each line like it was previous the change, practically you will need to move the block comment as a whole. It should be done for each file that is modified.

Reply:
This is resolved, eslint changed from tab to spaces, not indentation, that's why is weird visually on github.